lib/Test/Nginx/Socket.pm

```diff
@@ -1261,6 +1262,64 @@ sub check_error_log ($$$$) {
     }
 }
 
+sub check_shutdown_error_log ($) {
+    my ($block) = @_;
+    my $name = $block->name;
+    my $lines;
+
+    if (defined $block->shutdown_error_log) {
+        my $pats = $block->shutdown_error_log;
+
+        if (!ref $pats) {
+            chomp $pats;
+            my @lines = split /\n+/, $pats;
+            $pats = \@lines;
+
+        } elsif (ref $pats eq 'Regexp') {
+            $pats = [$pats];
+
+        } else {
+            my @clone = @$pats;
+            $pats = \@clone;
+        }
+
+        $lines ||= error_log_data();
+        #warn "error log data: ", join "\n", @$lines;
+        for my $line (@$lines) {
+            for my $pat (@$pats) {
+                next if !defined $pat;
+
+                if (ref $pat && $line =~ /$pat/ || $line =~ /\Q$pat\E/) {
+                    SKIP: {
```

Comment:
This is wrong since we cannot skip these tests unconditionally, otherwise these tests will *always* be skipped.